cmake/ospray_cmake_config/osprayConfig.cmake.in

```diff
@@ -243,3 +243,16 @@ set(OSPRAY_USE_FILE ${OSPRAY_CMAKE_ROOT}/osprayUse.cmake)
 include(FindPackageMessage)
 find_package_MESSAGE(ospray "Found OSPRay: ${OSPRAY_LIBRARY}"
   "[${OSPRAY_INCLUDE_DIRS}]")
+
+# Include modern cmake targets
+if (NOT CMAKE_VERSION VERSION_LESS "3.0")
+  foreach(f @OSPRAY_TARGET_LIST@)
+    # include all the target files
+    include("${OSPRAY_CMAKE_ROOT}/ospray-${f}-targets.cmake")
+    # Note (Qi): this is actually not necessary, but this prevents compilation errors when someone
+    #   used 'include_directories' by accidents.
```

Comment:
Actually I was wrong in the note. The only reason I added OSPRAY_INCLUDE_DIRS explicitly was because in ospray's implementation, `include_directories` is widely used instead of `target_include_directories` ... I wasn't sure if ospray wants such a big change ... @jeffamstutz